### DISCUSSION TITLE
Can't set box shadow color with tailwind v3.3.3

### DISCUSSION THREAD

**Discussion by hodge-jai** [tailwindlabs/tailwindcss#12227] [Help]
Hey there,

I've been trying to set the box shadow color on a div with these utility classes:

`<div className="rounded-lg bg-white mb-3 shadow-lg lg:shadow-lg lg:shadow-red-500">`

The shadow appears and is the appropriate size, however the color remains unchanged(i.e. it is the default color from the `shadow-lg` directive).  Not sure what is preventing the color from being set. Can anyone help me out?

My tailwind.config.js:

```
/** @type {import('tailwindcss').Config} */

const colors = require("tailwindcss/colors");

module.exports = {
  content: [
    "./app/**/*.{js,ts,jsx,tsx}",
    "./pages/**/*.{js,ts,jsx,tsx}",
    "./components/**/*.{js,ts,jsx,tsx}",
  ],

  theme: {
    extend: {

      minHeight: {
        almost: "calc(100vh - 40vh)",
      },
      colors: {
        sky: colors.sky,
        teal: colors.teal,
        cyan: colors.cyan,
        rose: colors.rose,
      },

    },
  },
  plugins: [
    require("@tailwindcss/forms"),
    require("@tailwindcss/line-clamp"),
    require("@tailwindcss/typography"),
  ],
};
```

**Comment by wongjn** [comment]
Consider checking you are looking at the element in a viewport wider than `1024px` in width. It seems to work fine in [Tailwind Play](https://play.tailwindcss.com/fVzwyHF41a?size=1180x720).

**Reply by hodge-jai** [reply]
Yup, I definitely am. I've also tried `shadow-red-500` without any breakpoint modifiers to no avail. The utility class doesn't appear in the dev console either.
<img width="521" alt="Screenshot 2023-10-17 at 2 18 50 PM" src="https://github.com/tailwindlabs/tailwindcss/assets/38635538/6d589cdf-4a96-4bf7-801b-d5ecdbf63d1e">

**Reply by wongjn** [reply]
Consider checking:
- Tailwind has compiled since you added the class.
- The `content` file globs sufficiently cover the file that the code appears in.
- [String concatenation/interpolation is not used to build the class name](https://tailwindcss.com/docs/content-configuration#dynamic-class-names). 

Otherwise, consider providing a remotely-hosted git repository that reproduces the unexpected behavior.

**Reply by hodge-jai** [reply]
> The content file globs sufficiently cover the file that the code appears in.

Could you elaborate more on what this entails?

**Reply by wongjn** [reply]
Consider reading [the documentation on configuring `content` properly](https://tailwindcss.com/docs/content-configuration).

**Comment by alex-krasikau** [comment]
Hello @hodge-jai!

1. Can you please specify the full path to the file where you are experiencing this issue, starting from the root of the project (e.g., `src/ui/Card.tsx`)?
2. Can you please try using these utilities in any file within any of these folders: `app`, `pages`, `components`, and let us know if you still encounter this issue?
